docs/contributors/release.md

```diff
@@ -1,162 +1,142 @@
-2. If you were releasing the `5.0.0` release candidate, label it `v5.0.0-rc.1`.
-3. The GitHub release screen should look like this:
-[![GitHub Release Screenshot](https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/reference/release-screenshot.png)](https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/reference/release-screenshot.png)
-4. Creative emojis related to a key feature in this release are encouraged. Emojis are fun!
-5. Publish the release.
+##### Bumping the Version and Tagging the Release
 
-### Build the Plugin
+1. Checkout the `release/x.x` branch.
+2. Create [a commit like this](https://github.com/WordPress/gutenberg/pull/13125/commits/13fa651dadc2472abb9b95f80db9d5f23e63ae9c), bumping the version number in `gutenberg.php`, `package.json`, and `package-lock.json` to `x.x.0-rc.1`.
+3. Create a Pull Request from the release branch into `master` using the changelog as a description and ensure the tests pass properly.
+4. Tag the RC version. `git tag vx.x.0-rc.1` from the release branch.
+5. Push the tag `git push --tags`.
+6. Merge the version bump pull request and avoid removing the release branch.
+
+##### Build the Plugin
 
 1. Run `git fetch --tags`.
-2. Check out the tag for this release; it will be the tag you created on the GitHub release page; assuming version `5.0.0`, you should run `git checkout v5.0.0-rc.1`.
+2. Check out the tag for this release, you should run `git checkout vx.x.0-rc.1`.
 3. Run `./bin/build-plugin-zip.sh` from the root of project. This packages a zip file with a release build of `gutenberg.zip`.
 
-### Publish the Release on GitHub
+##### Publish the Release on GitHub
 
-1. Upload the a `gutenberg.zip` file to the [GitHub Releases page](https://github.com/WordPress/gutenberg/releases)
-2. Post a link to the release page to the [`#core-editor` channel](https://wordpress.slack.com/messages/C02QB2JS7).
+1. [Create a new release on GitHub](https://github.com/WordPress/gutenberg/releases/new).
+2. If you were releasing the `x.x.0-rc.1` release candidate, label it `x.x.0-rc.1` and use the `vx.x.x-rc.1` as a tag.
+3. Upload the a `gutenberg.zip` file into the release.
+4. Use the changelog as a description of the release.
+5. Publish the release.
 
-Here's an example [release candidate page](https://github.com/WordPress/gutenberg/releases/tag/v3.8.0-rc.1); yours should look like that when you're finished.
+Here's an example [release candidate page](https://github.com/WordPress/gutenberg/releases/tag/v4.6.0-rc.1); yours should look like that when you're finished.
 
-### Commit to the Plugin Repository
+##### Publishing the Call For Testing
 
-You'll need to use Subversion for this step.
+Ping someone from the `[#core-test](https://wordpress.slack.com/messages/C03B0H5J0)` team to publish a call for testing post. Here's an [example call for testing post.](https://make.wordpress.org/test/2019/01/04/call-for-testing-gutenberg-4-8/)
```

Comment:
The inline code formatting prevents it from being shown as a link. The backticks should be inside the square brackets.